### PR TITLE
Remove Pipeline and Test Regression Rules & Fix Inaccuracies

### PR DESCRIPTION
**Pull request by markazerdd**
### What does this PR do? What is the motivation?

Removes references to the Pipeline and Test Performance Regression Quality Gate Rule Types and fixes other miscellaneous inaccuracies 

### Merge instructions

Merge queue is enabled in this repo. To have it automatically merged after it receives the required reviews, create the PR (from a branch that follows the `<yourname>/description` naming convention) and then add the following PR comment:

```
/merge
```

### CODE REVIEW

File: content/en/quality_gates/_index.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Quality Gates is in Preview.
 
 Quality Gates allow you to control software quality by configuring rules to block substandard code from deployment. You have control over what is merged into the default branch and deployed to production, and can ensure that the code running in production adheres to high quality standards, reducing incidents and minimizing unwanted behaviors.
 
-{{< img src="quality_gates/setup/pipeline_rule_1.png" alt="A pipeline rule that fails when code coverage for PCT is below or equal to zero in Quality Gates" style="width:100%" >}}
+{{< img src="quality_gates/setup/sca_2.png" alt="An SCA rule that fails any library vulnerabilities with critical or high severity are detected in the repository" style="width:100%" >}}
```
Comment (drichards-87) [inline content/en/quality_gates/_index.md:38]:
```suggestion
{{< img src="quality_gates/setup/sca_2.png" alt="An SCA rule that triggers a failure if any library vulnerabilities with critical or high severity are detected in the repository." style="width:100%" >}}
```

File: content/en/quality_gates/setup/_index.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ further_reading:
 
 To use Datadog Quality Gates, you can define one or more rules on the [**Quality Gate Rules** page][2] and integrate them in your CI pipeline with the [`datadog-ci gate evaluate` command][4].
 
-{{< img src="quality_gates/rules_list_1.png" alt="Quality Gates page in Datadog" style="width:100%" >}}
+{{< img src="quality_gates/rules_list_2.png" alt="Quality Gates page in Datadog" style="width:100%" >}}
```
Comment (drichards-87) [inline content/en/quality_gates/setup/_index.md:18]:
```suggestion
{{< img src="quality_gates/setup/rules_list_2.png" alt="Quality Gates page in Datadog" style="width:100%" >}}
```

Comment (drichards-87) [reply content/en/quality_gates/setup/_index.md:18]: The image link is broken. I believe this is the correct path to the image.

File: content/en/quality_gates/search/_index.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ further_reading:
 
 The [**Quality Gate Rules** page][1] is useful for developers who want to keep an eye on the quality gates for their build pipelines. You can see all of the rules defined by the organization.
 
-{{< img src="quality_gates/rules_list_1.png" alt="Quality Gate rules in Datadog displaying the rule name, evaluation, scope, blocking status, last modified date, and the creator avatar" style="width:100%" >}}
+{{< img src="quality_gates/rules_list_2.png" alt="Quality Gate rules in Datadog displaying the rule name, evaluation, scope, blocking status, last modified date, and the creator avatar" style="width:100%" >}}
```
Comment (drichards-87) [inline content/en/quality_gates/search/_index.md:18]:
```suggestion
{{< img src="quality_gates/setup/rules_list_2.png" alt="Quality Gate rules in Datadog displaying the rule name, evaluation, scope, blocking status, last modified date, and the creator avatar" style="width:100%" >}}
```

Comment (drichards-87) [reply content/en/quality_gates/search/_index.md:18]: The image link is broken. I believe this is the correct path to the image.

File: content/en/quality_gates/_index.md [hunk 4]
```diff
@@ -106,18 +96,18 @@ To create a Quality Gate rule, see the [Setup documentation][2].
 
 You can evaluate and update quality control processes by accessing Quality Gates rules on the [**Quality Gates Rules** page][6]. Improve your deployment practices based on your project requirements and desired performance outcomes. 
 
-{{< img src="quality_gates/rules_list_1.png" alt="List of Quality Gate rules in Datadog" style="width:100%" >}}
+{{< img src="quality_gates/rules_list_2.png" alt="List of Quality Gate rules in Datadog" style="width:100%" >}}
```
Comment (drichards-87) [inline content/en/quality_gates/_index.md:99]:
```suggestion
{{< img src="quality_gates/setup/rules_list_2.png" alt="List of Quality Gate rules in Datadog" style="width:100%" >}}
```

Comment (drichards-87) [reply content/en/quality_gates/_index.md:99]: The image link is broken. I believe this is the correct path to the image.

**Review by drichards-87 — CHANGES_REQUESTED** [review]
Left some minor feedback on the PR.

File: content/en/quality_gates/_index.md (hunk 4 above)
Comment (markazerdd) [reply content/en/quality_gates/_index.md:99]: Ah oops, I put the image file in the wrong place! I'm going to reject this suggestion and instead move the file to where I originally meant to put it

File: content/en/quality_gates/search/_index.md (hunk 3 above)
Comment (markazerdd) [reply content/en/quality_gates/search/_index.md:18]: Same comment here, I'll move the file to the correct location

**Review by drichards-87 — APPROVED** [review]

**Pull request merged**